lms/djangoapps/certificates/tests/test_webview_views.py

```diff
@@ -947,6 +968,55 @@ def test_html_view_certificate_available_date_for_instructor_paced_courses(self,
         )
         self.assertContains(response, date)
 
+    @override_settings(FEATURES=FEATURES_WITH_CERTS_ENABLED)
+    @ddt.data(
+        (True, False),
+        (False, False),
+        (True, True),
+        (False, True)
+    )
+    @ddt.unpack
+    def test_html_view_certificate_display_date(self, self_paced, date_override):
+        """
+        Test certificate web view should display the correct date on the
+        certificate in all cases:
+            * self-paced, no date override
+            * instructor-paced with certificate_available_date
+            * self-paced with date override
+            * instructor-paced with date override
+        """
+        self.course.self_paced = self_paced
+        if date_override:
+            self._add_certificate_date_override(datetime.datetime(2021, 5, 11))
+            self.cert.save()
```

Comment:
(Wups. This `self.cert.save()` is unnecessary here because it’s saved in `_add_certificate_date_override`; forgot to take it out. It still fails either way.